locales/tl-PH.json

```diff
@@ -0,0 +1,630 @@
+{
+  "a11y": {
+    "loading_page": "Nagloload ng pahina, mangyaring maghintay",
+    "loading_titled_page": "Nagloload ng pahinang {0}, mangyaring maghintay",
+    "locale_changed": "Nabago ang wika sa {0}",
+    "locale_changing": "Nagbabago ng wika, mangyaring maghintay",
+    "route_loaded": "Naloload na ang pahina {0}"
+  },
+  "account": {
+    "avatar_description": "Avatar ni {0}",
+    "blocked_by": "Ikaw ay na-block ng user na ito.",
+    "blocked_domains": "Mga na-block na domains",
+    "blocked_users": "Mga na-block na users",
+    "blocking": "Naka-block",
+    "bot": "BOT",
+    "favourites": "Mga paborito",
+    "follow": "Sundan",
+    "follow_back": "Sundan balik",
+    "follow_requested": "Humiling ng pagsunod",
+    "followers": "Mga tagasunod",
+    "followers_count": "{0} Mga tagasunod|{0} Tagasunod|{0} Mga tagasunod",
+    "following": "Mga sinusundan",
+    "following_count": "{0} Sinusundan",
+    "follows_you": "Nagsusunod sa'yo",
+    "go_to_profile": "Pumunta sa profile",
+    "joined": "Sumali",
+    "moved_title": "nagpahiwatig na ang kanilang bagong account ay:",
+    "muted_users": "Mga na-mute na users",
+    "muting": "Naka-mute",
+    "mutuals": "Mga mutual",
```

Comment:
In this case kabayan, since "mutuals" has become a common term in the Tagalog-speaking online community, it may be best to use the term as is without translation.  Para sa akin lang 😅